example/hello_object_detection_ssd_node/README.md

```diff
@@ -0,0 +1,77 @@
+# A Node.js example of object detection ssd using inference-engine-node
+
+### Install
+
+```sh
+$ npm install
+```
+
+### Usage
+
+```sh
+$ node main.js --help
+
+Hello Object Detection SSD
+
+  An example of object detection ssd using inference-engine-node.
+
+Options
+
+  -h, --help                Show this help message and exit.
+  -m, --model string        Required. Path to an .xml file with a trained model.
+  -i, --image string        Required. Path to an image file.
+  -d, --device string       Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL, MYRIAD
+                            or HETERO: is acceptable. Default value is CPU.
+  -n, --iterations number   Optional. The number of iterations for inference. Default value is 1.
+  -t, --threshold number    Optional. The minimal probability of the visibility object. Default value is 0.5.
+  -s, --sync                Optional. Specify to inference synchronously or asynchronously. Default value
+                            is false.
+```
+
+### Run
+
+For example on Windows, run MobileNet SSD on CPU plugin for 10 iterations:
+```sh
+$ node main.js -m ..\..\models\mobilenetssd\FP32\mobilenetssd.xml -i test.png -d CPU -n 10
```

Comment:
Two nits:
 - The model files are still under FP16 folder. Please rename that folder.
 - The image file should be test.jpg instead of test.png. Please change that.